sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample10_PublishAnEventBatchWithCustomSizeLimit.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Samples.Infrastructure;
+
+namespace Azure.Messaging.EventHubs.Samples
+{
+    /// <summary>
+    ///   An example of publishing events using a custom size limitation with the batch.
+    /// </summary>
+    ///
+    public class Sample10_PublishAnEventBatchWithCustomSizeLimit : IEventHubsSample
+    {
+        /// <summary>
+        ///   The name of the sample.
+        /// </summary>
+        ///
+        public string Name { get; } = nameof(Sample10_PublishAnEventBatchWithCustomSizeLimit);
+
+        /// <summary>
+        ///   A short description of the sample.
+        /// </summary>
+        ///
+        public string Description { get; } = "An example of publishing events using a custom size limitation with the batch.";
```

Comment:
Should we replace `{ get; } =` with `=>` like in other samples?

Reply:
Yes, we should.   Copy/paste magic!